rgbd/include/rgbd/Client_RGBD.h

```diff
@@ -0,0 +1,84 @@
+/**
+ * This client listens to RGBD messages on a single topic directing rgbd::Image.
+ */
+
+#ifndef RGBD_CLIENT_RGBD_H_
+#define RGBD_CLIENT_RGBD_H_
+
+#include <ros/node_handle.h>
+#include <ros/subscriber.h>
+#include <ros/callback_queue.h>
+
+#include "rgbd/types.h"
+
+#include "rgbd_msgs/RGBD.h"
+
+
+namespace rgbd {
+
+/**
+ * @brief Client which subscribes to RGBD topic
+ */
+class ClientRGBD {
```

Comment:
I don't find this class name too descriptive. I guess they're all RGBD clients: one using shared memory, one using 'regular ROS topics', and this one. Maybe `ClientRosRbgd`?

Reply:
I would like to keep this name, as there is also a `ServerRGBD`. Plus this is a ROS packages, so using ROS is 'default' and the shared memory is the exception.